### PR TITLE
Feature new gantt vue3

### PR DESCRIPTION
**Pull request by gaetanHerve**
Based on #635 
Integrates #648 

# Upgrade vue2 -> vue3

# Feature Kanban board
- create tasks and categories
- drag and drop tasks through categories
- edit tasks content (details, dates, assignee, attached file, visibility)
 
kanban_board_example

kanbanboard_taskdetails_example

# Feature new Gantt
- classic gantt interface for vue3 based on [@infectoone/ganttastic](https://github.com/InfectoOne/vue-ganttastic)
-  displays projects view or tasks view
- view tasks content

gantt_tasks_example

gantt_taskdetails_example

# Features not integrated to this PR (to plan for future PRs)
- project report
- projects warning delay display
- user authorisations by project
- task modification tracking

### CODE REVIEW

File: Modules/services/Model/SeTask.php [hunk 1]
```diff
@@ -0,0 +1,191 @@
+<?php
+
+require_once 'Framework/Model.php';
+
+/**
+ * Class defining projects tasks
+ *
+ * @author Sylvain Prigent
```
Comment (osallou) [inline Modules/services/Model/SeTask.php:8]: not the author....

File: Modules/services/Model/SeTaskCategory.php [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<?php
+
+require_once 'Framework/Model.php';
+
+/**
+ * Class defining the tracking sheet for se_projects
+ *
+ * @author Sylvain Prigent
```
Comment (osallou) [inline Modules/services/Model/SeTaskCategory.php:8]: author....

File: Modules/services/Model/ServicesInstall.php [hunk 3]
```diff
@@ -59,7 +59,12 @@ public function createDatabase() {
         
         $modelCabinet = new StockCabinet();
         $modelCabinet->createTable();
-        
+
+        $modelTask = new SeTask();
```
Comment (osallou) [inline Modules/services/Model/ServicesInstall.php:63]: missing import at top for Setask and SeTaskCategory

File: Modules/services/Model/SeProject.php [hunk 4]
```diff
@@ -24,31 +24,27 @@ public function __construct() {
...
+            `in_charge` int(11) NOT NULL DEFAULT 0,
+            `samplereturn` text NOT NULL DEFAULT '',
+            `samplereturndate` DATE,
+            `id_sample_cabinet` int(11) NOT NULL DEFAULT 0,
+            `samplesstocked` int(1) NOT NULL DEFAULT 0,
+            `samplescomment` text NOT NULL DEFAULT '',
+            PRIMARY KEY (`id`)
 		);";
         $this->runRequest($sql);
-        $this->addColumn('se_project', 'id_origin', 'int(11)', 0);
```
Comment (osallou) [inline Modules/services/Model/SeProject.php:43]: faut les garder, pour les upgrades lorsque la table existe (encore le cas pour nantes par ex)

**Review by osallou — COMMENTED** [review]
cf cas upgrade

File: Modules/services/Model/SeProject.php (hunk 4 above)
Comment (gaetanHerve) [reply Modules/services/Model/SeProject.php:43]: ok, j'avais mal interprété ton message à ce sujet

**Pull request merged**